### PR TITLE
add ttl to merchant redirect QRs

### PR DESCRIPTION
**Pull request by kaffikaffi**

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -283,6 +283,14 @@ components:
           pattern: '^https:\/\/[\w\.]+([\w#!:.?+=&%@\-\/]+)?$'
           example: 'https://www.example.com/myProduct'
           nullable: true
+        ttl:
+          type: integer
+          description: Time To Live in seconds (default is -1, which is infinate)
```
Comment (Sigvedash) [inline docs/swagger.yaml:288]: If no value.... infinate

File: docs/swagger.yaml [hunk 2]
```diff
@@ -283,6 +283,13 @@ components:
           pattern: '^https:\/\/[\w\.]+([\w#!:.?+=&%@\-\/]+)?$'
           example: 'https://www.example.com/myProduct'
           nullable: true
+        ttl:
+          type: integer
+          description: Time To Live in seconds (default is infinate)
```
Comment (khenrichsen) [inline docs/swagger.yaml:288]: Consider:
`Time to live in seconds. Default is infinite (-1)`

Comment (khenrichsen) [reply docs/swagger.yaml:288]: Okay. What about?
`Optional time-to-live field, given in seconds`

File: docs/swagger.yaml [hunk 3]
```diff
@@ -307,6 +314,10 @@ components:
           description: The target url of the QR (redirect destination)
           pattern: '^https:\/\/[\w\.]+([\w#!:.?+=&%@\-\/]+)?$'
           example: 'https://www.example.com/myProduct'
+        expiresIn:
+          type: integer
+          description: Time in seconds until expiration
```
Comment (khenrichsen) [inline docs/swagger.yaml:319]: Time in seconds until expiration. -1 means no expiration (infinite QR code)

**Comment by khenrichsen** [comment]
You could also add a small sentence about this in the readme

**Comment by khenrichsen** [comment]
Update document and api version by 0.0.1

**Review by khenrichsen — APPROVED** [review]

**Pull request merged**